storage/innobase/xtrabackup/utils/percona-xtrabackup-8.0_builder.sh

```diff
@@ -49,7 +49,6 @@ append_arg_to_args () {
             --build_deb=*) DEB="$val" ;;
             --get_sources=*) SOURCE="$val" ;;
             --build_tarball=*) TARBALL="$val" ;;
-            --branch=*) BRANCH="$val" ;;
```

Comment:
as I can see the BRANCH variable is used in the code

Reply:
Yes, deleted it since the BRANCH variable was duplicated